terraform/modules/backend/main.tf

```diff
@@ -0,0 +1,190 @@
+#tfsec:ignore:aws-s3-block-public-policy:block_public_policy is set to true, this is a false flag
+#tfsec:ignore:aws-s3-enable-bucket-encryption:using AES256 encryption with key managed by Amazon S3 instead of KMS
+#tfsec:ignore:aws-s3-ignore-public-acls:ignore_public_acls is set to true, this is a false flag
+#tfsec:ignore:aws-s3-no-public-buckets:restrict_public_buckets is set to true, this is a false flag
+#tfsec:ignore:aws-s3-encryption-customer-key:using key managed by S3 because cloudposse backend module can't be configured with KMS
+#tfsec:ignore:aws-s3-enable-versioning:versioning not required for replica of a bucket with versioning enabled
+#tfsec:ignore:aws-s3-enable-bucket-logging:exp:2023-07-24:ignoring this warning temporarily while deciding if logging is necessary
+#tfsec:ignore:aws-s3-specify-public-access-block:aws_s3_bucket_public_access_block is being used, this is a false flag
+module "tf_state_replica_bucket" {
+  providers = { aws = aws.ireland }
+  #checkov:skip=CKV_TF_1:providing git source with commit hash causes filename too long errors on Checkov, and we provide the version
+  #checkov:skip=CKV_AWS_273:iam user for bucket access isn't required nor created by cloudposse with our configuration, so we don't need to ensure this is controlled through SSO instead
+  #checkov:skip=CKV_AWS_300:lifecycle configuration is set below for aborting failed uploads, looks like a false flag
+  #checkov:skip=CKV2_AWS_34:use of ssm to store a parameter for the access key of an iam user isn't required nor created by cloudposse with our configuration, so we don't need to ensure encryption of the parameter
+  #checkov:skip=CKV2_AWS_62:we don't require event notifications on this bucket as it's only replicating the tfstate files as a fallback
+  source = "cloudposse/s3-bucket/aws"
+  # Cloud Posse recommends pinning every module to a specific version
+  version = "3.1.2"
+
+  acl                          = null
+  allow_encrypted_uploads_only = true
+  allow_ssl_requests_only      = true
+  block_public_acls            = true
+  block_public_policy          = true
+  bucket_name                  = var.state_replication_bucket_name
+  enabled                      = true
+  environment                  = "meta"
+  force_destroy                = false
+  ignore_public_acls           = true
+  lifecycle_configuration_rules = [{
+    abort_incomplete_multipart_upload_days = 1
+    enabled                                = true
+    expiration                             = {}
+    filter_and                             = {}
+    id                                     = "remove-incomplete-uploads"
+    noncurrent_version_expiration          = {}
+    noncurrent_version_transition          = []
+    transition                             = []
+  }]
+  restrict_public_buckets = true
+  s3_object_ownership     = "BucketOwnerEnforced"
+  s3_replication_enabled  = false
+  sse_algorithm           = "AES256"

```

Comment:
The encryption algorithm is configurable here, but we didn't think it made sense to have a customer-managed key here given that the backend module doesn't allow that for the main state bucket this is replicating.